service/src/main/frontend/src/functions/csvWrapper.ts

```diff
@@ -0,0 +1,12 @@
+// Called by exportTable to format the CSV
+// This is how Rows are Parsed and Formatted in the CSV
+// This may be changed to customize or change if a user would like to change/add data in the CSV
+export function wrapCsvValue(val?: any, formatFn?: any, row?: any) {
```

Comment:
Yes it is defined. It is an optional format function that can change how each of the rows are formatted. 
This function was used in the Quasar Docs to export CSVs as it has its own unique way of doing so. I decided to follow that idea and leave `formatFn` in here in case in the future someone wants to change _how_ the rows export.